runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -695,11 +695,21 @@ class INTERPRETER_CLASS
 	VMINLINE J9Method*
 	j2iVirtualMethod(REGISTER_ARGS_LIST, j9object_t receiver, UDATA interfaceVTableIndex)
 	{
+		J9Method *method;
 		void* const jitReturnAddress = VM_JITInterface::peekJITReturnAddress(_currentThread, _sp);
 		UDATA jitVTableOffset = VM_JITInterface::jitVTableIndex(jitReturnAddress, interfaceVTableIndex);
-		UDATA vTableOffset = sizeof(J9Class) - jitVTableOffset;
-		J9Class *clazz = J9OBJECT_CLAZZ(_currentThread, receiver);
-		return *(J9Method**)((UDATA)clazz + vTableOffset);
+#if defined(J9VM_OPT_VALHALLA_NESTMATES)
```

Comment:
Now that I think about it, I believe this code is completely unncessary - there's no way for the JIT to reach the interpreter via a virtual call to a private method.  The JIT will necessarily have to statically dispatch in these cases.

Reply:
This is part of the direct dispatch done by PicBuilder when `invokevirtual` was unresolved at compile time, and the callee is not compiled at runtime